tools/querytee/proxy_endpoint_test.go

```diff
@@ -793,11 +780,20 @@ func (b *mockProxyBackend) Preferred() bool {
 }
 
 func (b *mockProxyBackend) ForwardRequest(_ *http.Request, _ io.ReadCloser) (time.Duration, int, []byte, *http.Response, error) {
+	statusCode := 200
+	if b.responseIndex >= len(b.fakeResponseLatencies) {
+		statusCode = 500
+	}
```

Comment:
[nit] We could set the status code on the response to 200 by default, then change it in the second `if b.responseIndex >= ...` block below - this would help keep the condition in one place